set.mm

```diff
@@ -61377,9 +61440,38 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
       ( cdm fdmd sseqtrid ) AEHDBFABCEGIJ $.
   $}
 
+  $( The image of a class is a subset of its codomain.  (Contributed by Glauco
```

Comment:
```suggestion
  $( A direct image of a function with domain and codomain is a subset of its codomain.  (Contributed by Glauco
```

Reply:
I will change the comment to "The image of a class under a function with domain and codomain is a subset of its codomain." I do not now if the term "direct image" is adequate here, at least it is not defined at this place.

Reply:
As you prefer.